packages/platforms/react-native/lib/retry-queue/file-based.ts

```diff
@@ -8,25 +8,18 @@ import type {
 import type Directory from './directory'
 import timestampFromFilename from './timestamp-from-filename'
 
-function getLatestSpan (body: DeliveryPayload): DeliverySpan | undefined {
-  let latestSpan: DeliverySpan | undefined
-  let biggestTimestamp: bigint | undefined
+function getLastSpan (body: DeliveryPayload): DeliverySpan | undefined {
+  let lastSpan: DeliverySpan | undefined
 
   for (const resourceSpan of body.resourceSpans) {
     for (const scopeSpan of resourceSpan.scopeSpans) {
       for (const span of scopeSpan.spans) {
-        if (biggestTimestamp === undefined || BigInt(span.endTimeUnixNano) > biggestTimestamp) {
-          latestSpan = span
-
-          // store the biggest timestamp separately as a bigint so we don't need
-          // to parse it on every iteration
-          biggestTimestamp = BigInt(latestSpan.endTimeUnixNano)
-        }
+        lastSpan = span
       }
     }
   }
```

Comment:
now we're not searching for the biggest timestamp, wouldn't this be more performance to get the very last span by index, or using a similar array method?

Reply:
Good point - updated to loop backwards and get the last span by index